cypress/e2e/hermioneAccount.spec.cy.js

```diff
@@ -0,0 +1,56 @@
+import { faker } from '@faker-js/faker';
+/// <reference types='cypress' />
+
+describe('Bank app', () => {
+  const accountNumber = '1001';
+  const startBalance = 5096;
+  const randomDeposit = faker.number.int({ min: 1000, max: 9999 });
+  const currentBalance = startBalance + randomDeposit;
+  const withdrawAmount = faker.number.int({ min: 1000, max: 5000 });
```

Comment:
Similar to the previous comment, use `faker.datatype.number` instead of `faker.number.int` to generate a random integer.